terraform/hcloud/tour.tf

```diff
@@ -0,0 +1,12 @@
+# Dlang Tour (tour.dlang.org)
+
+resource "hcloud_server" "tour" {
+  name        = "tour"
+  image       = "ubuntu-22.04"
+  location    = "fsn1"
+  server_type = "cx11"
```

Comment:
```suggestion
  server_type = "cpx11"
```

I suggest a tiny price increase (4.79EUR vs 4.19EUR) for double the vCPU cores + likely better 1T perf (AMD EPYC 2nd Gen vs Intel® Xeon® Gold):

![image](https://user-images.githubusercontent.com/3475150/169658782-948f93c9-f66d-4b1a-89c9-60b95710260d.png)